packages/ace-cli/src/index.js

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
```

Comment:
the shebang is no longer used in this file, since the main executable is in now in`ace-cli/bin/ace.js`